src/reset-password/ResetPasswordPage.jsx

```diff
@@ -150,13 +154,15 @@ ResetPasswordPage.defaultProps = {
   token: null,
   match: null,
   errors: null,
+  errorCode: null,
 };
 
 ResetPasswordPage.propTypes = {
   intl: intlShape.isRequired,
   resetPassword: PropTypes.func.isRequired,
   validateToken: PropTypes.func.isRequired,
   token_status: PropTypes.string,
+  errorCode: PropTypes.string,
```

Comment:
Why we are always defining a new prop for each error case, can’t we utilize a single one for all?

Reply:
I still encourage you to use existing status props instead of creating a new one for each case. I guess we can also refactor and combine the `token_status` and `status` as well into a single prop and use that for our purpose here as well.